lib/pymedphys/_dicom/constants/core.py

```diff
@@ -37,12 +37,6 @@
 
 from pymedphys._imports import pydicom
 
-# Many thanks to the Medical Connections for offering free
```

Comment:
Why are credits being deleted?   Was it moved?

Reply:
Credits aren't being deleted. This was actually duplicated within the `constants/uuid.py` file. So, this duplication was deleted and the other file was used instead.